eng/pipelines/templates/steps/npm-release-task.yml

```diff
@@ -47,3 +46,11 @@ steps:
         arguments: -pathToArtifacts ${{parameters.PathToArtifacts}} -accessLevel "public" -tag "${{parameters.Tag}}" -additionalTag "${{parameters.AdditionalTag}}" -registry ${{parameters.Registry}} -npmToken $(azure-sdk-npm-token)
         pwsh: true
       condition: succeeded()
+
+  - task: Powershell@2
+    displayName: Verify package tags
+    inputs:
+      targetType: filePath
+      filePath: eng/scripts/verify-npm-tags.ps1
+      arguments: -artifactName ${{parameters.Artifact.name}} -workingDirectory $(System.DefaultWorkingDirectory)
```

Comment:
Don't we want to pass in the tags we expect to be set for this version?

Reply:
no, we're doing the "determine the version" in the verify-npm-tags.ps1 script itself

Reply:
checks for all 3 potential tags (next, dev, latest)

Reply:
follow up comments below